cypress/integration/search-ui.spec.js

```diff
@@ -38,24 +38,27 @@ describe(`${playground} playground test`, () => {
 
   it('Sort by recommendationCound ascending', () => {
     const select = `.ais-SortBy-select`
-    cy.get(select).select('steam-video-games:recommendationCount:asc')
+    cy.get(select).select('games:recommendationCount:asc')
     cy.wait(1000)
     cy.get(HIT_ITEM_CLASS).eq(0).contains('Deathmatch Classic')
   })
 
   it('Sort by default relevancy', () => {
     const select = `.ais-SortBy-select`
-    cy.get(select).select('steam-video-games')
```

Comment:
Big PRs usually hide many essential things reviewers don't get because they feel overwhelmed by the changes.

This change seems to be just a simple index name, a harmless change, but if you add ten more changes like that, the git diff is too complex to be helpful.

Btw, no need to move this to a different PR or something like that, it is just something to be in mind.

Reply:
I agree, thanks for pointing this out :)